README.md

```diff
@@ -65,38 +65,52 @@ NOTE: OpenCL host side program should be linked with icd loader `-lOpenCL`.
 ```
 $ <pocl-install-dir>/bin/poclcc -l
 
+// The following output should be shown:
 LIST OF DEVICES:
 0:
   Vendor:   THU
     Name:   Ventus GPGPU device
  Version:   2.2 HSTR: THU-ventus-gpgpu
 ```
 
-Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example(Work in progress).
+Also, you can try to set `POCL_DEBUG=all` and run example under `<pocl-build-dir>` to see the full OpenCL software stack execution pipeline. For example:
```

Comment:
This example may not be appropriate. The comparison before and after the environment variable is imported is:    
![Screenshot from 2024-09-26 15-11-25](https://github.com/user-attachments/assets/8edb6f23-d44e-4585-be22-f9a8f6c531f7)

Reply:
Thanks for pointing it out! Now I change to use `vecadd`, the same example given in "4.4: Driver using example" and correct the statement.